components/dialog/stories/dialog.stories.js

```diff
@@ -4,46 +4,69 @@ import { isOpen } from "@spectrum-css/preview/types";
 import { Template as Typography } from "@spectrum-css/typography/stories/template.js";
 import { version } from "../package.json";
 import { DialogFullscreen, DialogFullscreenTakeover, DialogGroup } from "./dialog.test.js";
+import { Template } from "./template";
 
 /**
  * A dialog displays important information that users need to acknowledge. They appear over the interface and block further interactions.
+ * 
+ * The alert variants that were previously a part of Dialog were moved to their own component, [Alert Dialog](/docs/components-alert-dialog--docs).
  */
 export default {
 	title: "Dialog",
 	component: "Dialog",
 	argTypes: {
-		heading: { table: { disable: true } },
+		heading: {
+			name: "Heading",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "text" },
+		},
 		content: { table: { disable: true } },
-		footer: { table: { disable: true } },
 		hasFooter: {
 			name: "Has footer",
 			type: { name: "boolean" },
 			table: {
 				type: { summary: "boolean" },
-				category: "Component",
+				category: "Content",
 			},
 			control: "boolean",
 		},
-		isDismissable: {
-			name: "Dismissable",
-			type: { name: "boolean" },
+		footer: { table: { disable: true } },
+		size: {
+			name: "Size",
+			type: { name: "string", required: true },
 			table: {
-				type: { summary: "boolean" },
+				type: { summary: "string" },
 				category: "Component",
 			},
-			control: "boolean",
+			options: ["small", "medium", "large"],
+			control: "select",
+		},
+		layout: {
+			name: "Layout",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			options: ["default", "fullscreen", "fullscreenTakeover"],
```

Comment:
```suggestion
			options: ["fullscreen", "fullscreenTakeover"],
```

Propose removing the default option from the list and just make the absence of a setting drive the class.

Reply:
Based on our conversation in the breadcrumbs PR ([2793](https://github.com/adobe/spectrum-css/pull/2793#discussion_r1726039443)), I was going to keep this as is for now. It's a select drop down currently, and I added the `defaultValue` to the arg table. Does that all sound ok?

https://github.com/adobe/spectrum-css/pull/2833/commits/bb342743ca3ce29ad8e859859157e832c8356942